### ISSUE TITLE
OSX and Linux Support with PowerShell Core\6

### ISSUE THREAD

**Issue by jkdba** [PSKeePass/PoShKeePass#126]
This needs researching and and testing to see if this is even feasible.

**Comment by jkdba** [comment]
Did some initial research here, keepasslib makes references to windows forms mainly among others that will slow this down. Will have to look and consider .net core ported keepasslib options if any viable.

**Comment by JakeMoe** [comment]
FYI, not just OSX/Linux.  Tried this Windows PSCore (6.2.1) using a keyfile, and got:

New-Object : Exception calling ".ctor" with "1" argument(s): "Could not load type 'System.Security.Cryptography.ProtectedMemory' from assembly 'System.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'."
At C:\Users\JohnM\Documents\PowerShell\Modules\PoShKeePass\2.1.3.0\PoShKeePass.psm1:2636 char:39

and

Exception calling "Open" with "3" argument(s): "Could not load file or assembly 'System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified."
At C:\Users\JohnM\Documents\PowerShell\Modules\PoShKeePass\2.1.3.0\PoShKeePass.psm1:2664 char:9

**Comment by jkdba** [comment]
@JakeMoe that is correct, the keepass lib uses winforms amount other .Net Standard only features, Updated Title to reflect this.
